src/content/docs/discovery/index.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview
+description: Learn about Adobe Commerce on Edge Delivery Services project requirements, data validation, dropins, and Luma Bridge.
+---
+
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+
+Before starting any Adobe Commerce on Edge Delivery Services project, you must conduct a discovery phase to scope the project and ensure that there are no major roadblocks or risks.
+
+## Big picture
+
+In the [create your storefront](/get-started/) tutorial, you learned how to quickly create a new project based on the [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce). The boilerplate provides a starter storefront that uses a pre-configured Adobe Commerce environment.
+
+The next step is to plan, build, optimize, and launch a production-ready storefront connected to your own Adobe Commerce instance and storefront services. Here's an overview of the process:
+
+<Diagram caption="Storefront project planning and delivery process.">
+  ![Storefront project planning and delivery process.](@images/implementation/implementation-overview-v1.svg)
+</Diagram>
+
+<Callouts>
+
+1. **Discovery**: Understand the requirements and goals of the project.
+1. **Setup**: Configure the project environment and tools.
+1. **Marketing and SEO**: Optimize the storefront for search engines and marketing campaigns.
+1. **Launch**: Deploy the storefront to a production environment.
+
+</Callouts>
+
+The discovery phase is important because Adobe Commerce is a highly customizable platform with a large thid-party extension ecosystem. Migrating an Adobe Commerce storefront to Edge Delivery Services is similar to migrating to a headless storefront (like [PWA Studio](https://developer.adobe.com/commerce/pwa-studio)).
+
+:::note[Important]
+The key requirement for a headless storefront implementation is that all required data is provided by APIs. Adobe Commerce provides all of the API coverage that you need for a successful implementation.
+:::
+
+## Existing storefronts
+
+If you have an existing Adobe Commerce storefront (Luma or PWA Studio), you can migrate it to Edge Delivery Services. The migration process involves identifying:
```

Comment:
you have several options how you can approach storefront modernization with Edge delivery: 
 - progressive implementation: Selected parts of the commerce funnel are implemented as new storefront and the rest is reused from existing storefront. 
 - full implementation: the entire storefront delivered with Edge and existing storefront retired.
 
 Progressive adoption enables to unlock business value with Edge Delivery sooner, minimizing risk of migration. You can start from implementing home page only and reuse the rest of the storefront. Next step could be implementing catalog with product listing, search and product details and reusing cart, checkout and account from the existing storefront. 
 This approach comes with the cost of maintaining 2 storefronts in parallel, so before you choose, assess which approach is right for you. These are the key factors to consider:
 
-- ** Business metrics to improve** :  what are the areas on your storefront  drive business performance. What are the challenges based on the site analytics you see. 
--  ** Level of storefront customization**: how many experiences are built custom on your storefront. Do they rely on custom business logic? 
-- **Third-party extensions**: how many third-party extensions  are you using in your storefront. For example, do you need to connect to third-party services?
-- ** B2B**: is there any Adobe Commerce B2B modules enabled in your storefront 
 
We recommend Progressive implementation if any of these correct:

- The main business metrics are  brand visibility in search engines, customer acquisition cost and customer engagement 
- Current storefront is heavily customized in  areas of checkout and user account 
- There are many 3rd party integrations in the current storefront that affect transactional flow
- Storefront supports B2B use cases with Adobe Commerce modules 

The full implementation is a better option if:

- Business objectives include improving customer conversion and reengagement 
- Current checkout flow relies only on few 3rd party integrations, e,g, payments, shipping, taxes
- No Adobe Commerce  B2B modules enabled 

### Progressive Implementation 

For the progressive implementation you will need to look into the following: